.travis.yml

```diff
@@ -24,5 +24,6 @@ before_script:
 script:
 # Unable to initialize database, use the following code to debug. - docker-compose up --build
 # docker-compose up --build
+- bash -c 'shopt -s globstar nullglob; GLOBIGNORE="vendor*:node_modules*"; shellcheck **/*.{sh,ksh,bash}'
```

Comment:
@gdevenyi Would you be able to add this to the `checkstatic` target in the `Make file`? Instead of directly in `.travis.yml`?

As you can see on the line below, Travis will run `make checkstatic` so the command will still be a part of our CI. This will also give the benefit the options of developers being able to easily run the command locally.

Reply:
@gdevenyi Would it be possible for you to address what @johnsaigle raise?

Reply:
Missed this request, done (hopefully edited the Makefile properly since I can't really test that command locally)